Content.Server/GameObjects/EntitySystems/Atmos/PlayerGasOverlay.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Content.Shared.GameObjects.EntitySystems.Atmos;
+using Robust.Shared.Map;
+
+namespace Content.Server.GameObjects.EntitySystems.Atmos
+{
+    public sealed class PlayerGasOverlay
```

Comment:
This is a public class but it doesn't actually seem to be exposed by `GasTileOverlaySystem`?